dstep/Configuration.d

```diff
@@ -49,6 +49,10 @@ struct Configuration
     @("public-submodules", "Use public imports for submodules [default].")
     bool publicSubmodules = false;
 
+    // rename imports of submodules to D conforming form
+    @("normalize-submodules", "Rename imports of submodules to D conforming form [default].")
```

Comment:
What is `submodules` referring to? Shouldn't it be `module`?

Reply:
`normalize-modules`?
`rename-module-names`?
`rename-modules`?

Reply:
`normalize-modules` sounds better.